### PR TITLE
api: Revert for showing State in StoreInfo

### PR DESCRIPTION
**Pull request by CabinfeverB**
### What problem does this PR solve?
refer to https://github.com/pingcap/tiup/issues/1684

### What is changed and how it works?
Revert #4334  

### Release note

```release-note
None.
```

### CODE REVIEW

File: tests/cluster.go [hunk 1]
```diff
@@ -361,7 +361,7 @@ func (s *TestServer) GetStoreRegions(storeID uint64) []*core.RegionInfo {
 func (s *TestServer) BootstrapCluster() error {
 	bootstrapReq := &pdpb.BootstrapRequest{
 		Header: &pdpb.RequestHeader{ClusterId: s.GetClusterID()},
-		Store:  &metapb.Store{Id: 1, Address: "mock://1", LastHeartbeat: time.Now().UnixNano()},
```
Comment (rleungx) [inline tests/cluster.go:364]: ditto

File: server/api/trend.go [hunk 2]
```diff
@@ -130,8 +130,8 @@ func (h *trendHandler) getTrendStores() ([]trendStore, error) {
 	for _, store := range stores {
 		info := newStoreInfo(h.svr.GetScheduleConfig(), store)
 		s := trendStore{
-			ID:              info.Store.StoreID,
```
Comment (rleungx) [inline server/api/trend.go:133]: We can keep it.

**Review by rleungx — COMMENTED** [review]
The rest LGTM.

**Review by nolouch — APPROVED** [review]
lgtm

**Review by JmPotato — APPROVED** [review]

**Pull request merged**